docs/datascience/data-science-tutorial.md

```diff
@@ -117,9 +121,9 @@ This tutorial uses the [Titanic dataset](https://hbiostat.org/data/repo/titanic.
 
    ![Graphing the titanic data](images/data-science-tutorial/jupyter-cell-02.png)
 
-   To better view details on the graphs, you can open them in the plot viewer by hovering over the upper right corner of the graph and clicking the button that appears.
+   > **Tip**: To quickly copy your graph, you can hover over the upper right corner of your graph and click on the `Copy to Clipboard` button that appears. You can also better view details of your graph by clicking the `Expand image` button.
```

Comment:
"select the **Copy to Clipboard** button..."
"by selecting the **Expand image** button."